opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -366,11 +352,13 @@ def __init__(
         instrumentation_info: "InstrumentationInfo",
     ):
         self.instrumentation_info = instrumentation_info
-        self.batcher = UngroupedBatcher(source.stateful)
+        self.batcher = Batcher(source.stateful)
```

Comment:
Can we lose this whole class yet?

Reply:
I am not sure. SDK specs are still not out.

Reply:
We should feel free to move ahead of the spec, or this may never get written.